src/build/validator.md

```diff
@@ -79,28 +107,13 @@ Note that you can see the logs by running the command:
 sudo docker logs -f cudos-start-full-node-client-testnet-public-01
 ```
 
-#### Initialize and start the node without being a validator
```

Comment:
happy to remove these from this PR but please ensure this is re-added in the new Developer section